### ISSUE TITLE
Change planetary material percentages to percentiles

### ISSUE THREAD

**Issue by Javelias** [EDCD/EDDI#1917]
## EDDI version in which issue is found
3.7.0

## Expected
To apply distributions just like with other events so that the body materials script can choose the percentile to react towards. The net result will be higher accuracy and more flexibility on material scanning feedback.

For example, the default body material script can be adapted to react to the 99th percentile, the 95th, etc and the user can adapt it if wanted.

## Observed
Currently we work with a simple 10% and 25% cut off from the top record, excluding outliers.

## Investigation
This would likely involve parsing the entire body dump file.

**Comment by Tkael** [comment]
There's good info [here](https://docs.google.com/spreadsheets/d/1jdFFH7jMDC3Dh4B65eZlMhxyRRC8Eysk6xzUnxmORXQ/edit#gid=1591816466) for building statistical material distributions.

**Comment by Tkael** [comment]
Having reviewed the linked data, material distributions do not follow normal distributions. The distributions are skewed and a different predictive model is required. 

While I've tried to identify the correct distribution type and characterize the applicable parameters describing each distribution, I do not believe that I have the statistical prowess to do so with any degree of confidence. 

Closing as we do not have enough information and have no current plans to implement this.
